docs/pages/blog/mui-x-v7.md

```diff
@@ -0,0 +1,383 @@
+  </video>
+  <figcaption style="color: #888; font-size: 0.9em; text-align: center;">A before and after showcase, first with horizontal and then with vertical scrolling.</figcaption>
+</figure>
+
+:::warning
+The scrolling experience is far better, but the solution has a downside: **column headers can no longer be transparent** because content scrolls behind them.
+
+If you're using Material UI's theming system, the Data Grid will adapt to your application seamlessly.
+
+Without it, you may need to manually set an appropriate background color to maintain your design.
+:::
+
+### Improved columns panel design
+
+As part of an overarching project aimed at enhancing usability in managing columns within the Data Grid, we've redesigned the column visibility panel, introducing a more streamlined, checkbox-based interface to replace the previous toggle switch design.
+
+The goal is to further develop this panel by incorporating additional column management functions, such as column reordering and pinning. Additionally, We've extracted the internal component to ease the introduction of the upcoming pivoting UI.
+
+<figure style="padding:0; margin:0; margin-bottom:20px">
+  <img width="310" alt="The new column management panel design" src="/static/blog/mui-x-v7/column-management-panel.png">
+  <figcaption style="color: #888; font-size: 0.9em; text-align: center;">The new column managemeent panel design</figcaption>
+</figure>
+
+### Date Object support in filter model
+
+The `filterModel` now supports `Date` objects for `date` and `dateTime` column types, providing a more intuitive and efficient filtering experience.
+
+While string values remain compatible for these types, any updates to the `filterModel` made through the UI (such as via the filter panel) will automatically use `Date` objects, ensuring consistency and ease of data handling.
+
+### Smaller bundle size
+
+Introducing a separate entry point for locales has significantly reduced the bundle size of the barrel index when tree-shaking isn't operational (for example, Webpack in dev mode).
+
+As a reference, with the `@mui/x-data-grid` npm package, this change led to a reduction of approximately 19% – shrinking the bundle size from [114.2kB](https://bundlephobia.com/package/@mui/x-data-grid@6.19.2) to [92.9kB](https://bundlephobia.com/package/@mui/x-data-grid@7.0.0).
+
+<figure style="padding:0; margin:0">
+  <img width="600" alt="A chart showcasing the bundle size change" src="/static/blog/mui-x-v7/new-bundle-size.png" >
+  <figcaption style="color: #888; font-size: 0.9em; text-align: center;">A chart showcasing the Data Grid's bundle size change.</figcaption>
+</figure>
+
+### New stable features
+
+To ensure that new features can be introduced throughout the lifecycle of a major stable version without waiting for pre-releases, MUI X releases them marked as `experimentalFeatures` or with the `unstable_` prefix.
```

Comment:
Optional improvement: Although it's quoted in the subsequent warning too, but just to emphasize a bit more. Feel free to discard the suggestion.
```suggestion
To ensure that new features can be introduced throughout the lifecycle of a major stable version or any changes could be made to them after they are released without waiting for the product pre-releases, MUI X releases them marked as `experimentalFeatures` or with the `unstable_` prefix.
```